### DISCUSSION TITLE
Language not displayed in repository

### DISCUSSION THREAD

**Discussion by SidPaliwal** [github-linguist/linguist#5276] [Classification]
My repository [https://github.com/SidPaliwal/CryptoCurrency-Tracker](url) is not showing any language ? Can anyone please help resolve this issue !

**Comment by Nixinova** [comment]
[Your repo](https://github.com/SidPaliwal/CryptoCurrency-Tracker) is classified as Kotlin for me.

**Reply by SidPaliwal** [reply]
Yes its working now thanks

**Reply by lildude** [reply]
As with https://github.com/github/linguist/discussions/5278, the background job wasn't triggered when you initially pushed for some reason. A subsequent push has kicked off the job hence you now see the stats.
